Packs/MailListener/Integrations/MailListenerV2/MailListenerV2.py

```diff
@@ -409,7 +433,7 @@ def generate_search_query(time_to_fetch_from: Optional[datetime],
     return messages_query_list
 
 
-def test_module(client: IMAPClient) -> str:
+def script_test_module(client: IMAPClient) -> str:
```

Comment:
I changed thew name because if it starts with test_ the IDE will think it's a unit test